### ISSUE TITLE
init_db fails

### ISSUE THREAD

**Issue by spinus** [aartur/mschematool#4]
When I run init_db I have this:

```
    WHERE table_name=%s)""", [self.TABLE])
  File "lib/python3.4/site-packages/mschematool/executors/postgres.py", line 23, in execute
    log.info('Executing SQL: <<%s>>', core._simplify_whitespace(realsql))
  File "lib/python3.4/site-packages/mschematool/core.py", line 30, in _simplify_whitespace
    return ' '.join(s.split())
```

**Comment by aartur** [comment]
This is not a full stack trace?

I see you have added Python 3.4 as supported in your pull request, but it looks like it's the source of trouble here. (I have tested it under 2.7 only)

**Comment by spinus** [comment]
Yes, it's not full. Pasting here:

```
Traceback (most recent call last):
  File "bin/mschematool", line 11, in <module>
    sys.exit(main())
  File "/lib/python3.4/site-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/lib/python3.4/site-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/lib/python3.4/site-packages/click/core.py", line 936, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/lib/python3.4/site-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/lib/python3.4/site-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/lib/python3.4/site-packages/mschematool/cli.py", line 33, in init_db
    ctx.obj.migrations.initialize()
  File "/lib/python3.4/site-packages/mschematool/executors/postgres.py", line 48, in initialize
    WHERE table_name=%s)""", [self.TABLE])
  File "/lib/python3.4/site-packages/mschematool/executors/postgres.py", line 23, in execute
    log.info('Executing SQL: <<%s>>', core._simplify_whitespace(realsql))
  File "/lib/python3.4/site-packages/mschematool/core.py", line 30, in _simplify_whitespace
    return ' '.join(s.split())
TypeError: sequence item 0: expected str instance, bytes found
```

Mhm, that's true. I added python3 and only checked that python3 can run code without syntax errors, sorry.

**Comment by aartur** [comment]
I published 0.6.5 release which removes Python 3.4 classifier, but adds some Python 3.x compatibility (error you got should be fixed). Python 3.x should work for Postgres migrations. It won't work for Cassandra, because it uses libraries requiring 2.7.

Officially the tool requires Python 2.7.

**Comment by spinus** [comment]
Oh OK, my fault I didn't check cassandra, sorry for that. Thanks for quick fix and release.
